test/e2e/testdrivers/1.26.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 DigitalOcean
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+StorageClass:
+  FromName: true
+SnapshotClass:
+  FromName: true
+DriverInfo:
+  Name: dobs.csi.digitalocean.com-dev
+  SupportedSizeRange:
+    Max: 100Gi
+    Min: 1Gi
+  Capabilities:
```

Comment:
Just to confirm: have we validated that no new capabilities were added in the e2e test suite/framework that we may possibly support already?

Reply:
Discovered together that upstream testing didn't support any new capabilities that our driver would support already. (This is becoming increasingly unlikely now that the upstream tests support all baseline features.)